android/expoview/src/main/java/host/exp/exponent/notifications/PushNotificationHelper.java

```diff
@@ -164,7 +167,7 @@ public void onLoadBitmap(final Bitmap bitmap) {
 
         // Create notification object
         boolean isMultiple = mode == Mode.COLLAPSE && unreadNotifications.length() > 1;
-        final ReceivedNotificationEvent notificationEvent = new ReceivedNotificationEvent(experienceId, body, notificationId, isMultiple, true);
+        final ReceivedNotificationEvent notificationEvent = new ReceivedNotificationEvent(experienceKey.getStableLegacyId(), body, notificationId, isMultiple, true);
```

Comment:
`ReceivedNotificationEvent` now takes a scope key:
```suggestion
        final ReceivedNotificationEvent notificationEvent = new ReceivedNotificationEvent(experienceKey.getScopeKey(), body, notificationId, isMultiple, true);
```